tee-worker/ts-tests/examples/direct-invocation/index.ts

```diff
@@ -186,7 +200,7 @@ async function runDirectCall() {
 }
 
 (async () => {
-    await runDirectCall().catch((e) => {
+    await runDirectCall('sr25519').catch((e) => {
```

Comment:
Would you like to add a case for `ethereum`? This is quite important that we at least smoke-tested it

Reply:
Added

Unfortunately assertion fails even on `dev` branch

```
AssertionError: expected 1 to equal 2
    at runDirectCall (/home/kziemianek/projects/litentry/litentry-parachain/tee-worker/ts-tests/examples/direct-invocation/index.ts:126:12)
    at async /home/kziemianek/projects/litentry/litentry-parachain/tee-worker/ts-tests/examples/direct-invocation/index.ts:189:5 {
  showDiff: true,
  actual: 1,
  expected: 2,
  operator: 'strictEqual'
}
```

It looks like linking fails
```
pushing error event ... error: LinkIdentityFailed: StfError(BoundedVec([87, 114, 111, 110, 103, 87, 101, 98, 51, 78, 101, 116, 119, 111, 114, 107, 84, 121, 112, 101, 115], 100))
```

Reply:
I made a few refactoring and fixes in https://github.com/litentry/litentry-parachain/pull/1875 - at least it went through locally (for substrate case)